### PR TITLE
docs(api): add `<NuxtWelcome />` component docs

### LINKED ISSUE #6715: Document `<NuxtWelcome />`
It would be nice if the API docs included the `<NuxtWelcome />` component under the components tab. ([https://v3.nuxtjs.org/api/components/](https://v3.nuxtjs.org/api/components/nuxt-page)) This will let people know that this component exists, and provide a brief explanation of how it's used.

This may be a small change, but they add up in the end. Thanks for creating such a great framework!

### PR DESCRIPTION
**Pull request by umutiq**
### 🔗 Linked issue

resolves #6715 

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Solves #6715

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/content/3.api/2.components/6-nuxt-welcome.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# `<NuxtWelcome>`
+
+Nuxt uses the `<NuxtWelcome>` component to greet users when framework initialized. It provides information about latest version of framework, documentation link and how to start building quickly.
```
Comment (danielroe) [inline docs/content/3.api/2.components/6-nuxt-welcome.md:3]:
```suggestion
Nuxt uses the `<NuxtWelcome>` component to greet users in new projects. It includes links to the Nuxt documentation, source code and social media accounts.
```

**Comment by pi0** [comment]
Do you mind to add a `::ReadMore` link to https://github.com/nuxt/ui?

**Comment by umutiq** [comment]
@pi0 Since we didn't mention it's imported from @nuxt/ui, i think it's inapporpriate.

**Comment by danielroe** [comment]
Maybe we should mention it then.

**Comment by umutiq** [comment]
@danielroe I added alert and Readmore links to document. Can you check it?

File: docs/content/3.api/2.components/6-nuxt-welcome.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# `<NuxtWelcome>`
+
+::alert{type=info icon=🔎}
+This component is part of [@nuxt/ui](https://github.com/nuxt/ui)
+::
+
+Nuxt uses the `<NuxtWelcome>` component to greet users in new projects. It includes links to the Nuxt documentation, source code and social media accounts.
```
Comment (pi0) [inline docs/content/3.api/2.components/6-nuxt-welcome.md:7]:
```suggestion
Nuxt uses the `<NuxtWelcome>` component to greet users in new projects made from the starter template. It includes links to the Nuxt documentation, source code, and social media accounts.
```

**Review by pi0 — APPROVED** [review]
Thanks <3

File: docs/content/3.api/2.components/6-nuxt-welcome.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# `<NuxtWelcome>`
+
+::alert{type=info icon=🔎}
+This component is part of [@nuxt/ui](https://github.com/nuxt/ui)
```
Comment (DamianGlowala) [inline docs/content/3.api/2.components/6-nuxt-welcome.md:4]:
```suggestion
This component is part of [@nuxt/ui](https://github.com/nuxt/ui).
```

**Comment by BD103** [comment]
Thank you!

**Pull request merged**